ci-operator/step-registry/ipi/conf/aws/custom-az/ipi-conf-aws-custom-az-commands.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+REGION="${LEASED_RESOURCE}"
+CONFIG="${SHARED_DIR}/install-config.yaml"
+
+if [ ${REGION} == "us-east-1" ]; then
```

Comment:
I think we can introduce `ZONES_COUNT` env to some similar things like `ipi-conf-aws`, so that the step works with any selected region instead of the region and zone hardcode settings.
```
function join_by { local IFS="$1"; shift; echo "$*"; }

mapfile -t AVAILABILITY_ZONES < <(aws --region "${REGION}" ec2 describe-availability-zones | jq -r '.AvailabilityZones[] | select(.State == "available") | .ZoneName' | sort -u)
ZONES=("${AVAILABILITY_ZONES[@]:0:${ZONES_COUNT}}")
ZONES_STR="[ $(join_by , "${ZONES[@]}") ]"
echo "AWS region: ${REGION} (zones: ${ZONES_STR})"
```

Reply:
This is a good point I've updated and tested the solution you've proposed here and it is working.

[PR to run short test run to test the functionality
](https://github.com/openshift/release/pull/38123)
[job](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/38123/rehearse-38123-periodic-ci-rhpit-interop-tests-main-test-az-4.13-az-test-interop-aws/1646239759263076352)
[result of ref ipi-conf-aws-custom-az](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/38123/rehearse-38123-periodic-ci-rhpit-interop-tests-main-test-az-4.13-az-test-interop-aws/1646239759263076352/artifacts/az-test-interop-aws/ipi-conf-aws-custom-az/build-log.txt)